content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -7,7 +7,19 @@ tocDepth: 2
 
 <TopBlock>
 
-Referential actions determine what happens to a record when a related record is deleted or updated. In [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and later (before 3.0.0 with a preview feature flag `referentialActions`, afterwards without), you can define referential actions on your relation fields in your Prisma schema.
+Referential actions determine what happens to a record when your application deletes or updates a related record.
+
+From version 2.26.0, you can define referential actions on the relation fields in your Prisma schema. This allows you to do cascading deletes and cascading updates.
```

Comment:
Nitpicks, can be ignored:

Maybe better to point out that these are examples of what you can do with referential actions (as they are used for other things too)

Also should maybe be something like 'define cascading deletes and updates at a Prisma level', as they could already be done manually.

Reply:
Thanks, I've brought these suggestions into the doc.